edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/ModuleManagementHttpClientTest.cs

```diff
@@ -271,6 +273,33 @@ public async Task ModuleLogsTest(string serverApiVersion, string clientApiVersio
             Assert.Equal(buffer.Length, bytesRead);
         }
 
+        [Theory]
+        [MemberData(nameof(VersionDateTimeMap))]
+        public async Task ModuleLogsTestInvalidDateTime(string serverApiVersion, string clientApiVersion, string since, string until)
+        {
+            // Arrange
+            IModuleManager client = new ModuleManagementHttpClient(this.serverUrl, serverApiVersion, clientApiVersion);
+            // Act and Assert
+            var ex = await Assert.ThrowsAsync<ArgumentException>(async () =>
+            {
+                await client.GetModuleLogs("edgeHub", false, Option.Some<int>(10), Option.Some<string>(since), Option.Some<string>(until), Option.Some(true), CancellationToken.None);
+            });
+            Assert.Contains("The correct format should be yyyy-MM-dd'T'HH:mm:ssZ", ex.Message);
+        }
+
+        [Theory]
+        [MemberData(nameof(VersionMap))]
+        public async Task ModuleLogsValidDateTime(string serverApiVersion, string clientApiVersion)
```

Comment:
nit: There is a naming inconsistency between this and the above method. Here it is missing `Test` in the name.

Reply:
Done